### PR TITLE
Feat/external pools

### PR DESCRIPTION
**Pull request by ixm-759**

### CODE REVIEW

File: src/stats/stats.service.ts [hunk 1]
```diff
@@ -823,4 +820,44 @@ export class StatsService {
...
+
+  async getIncentivizedPools() {
+    const cachedValue = await this.cacheManager.get('pools');
+    if (cachedValue) {
+      this.logger.log('Hit getIncentivizedPools() cache');
+      return cachedValue;
+    }
+    const { data } = await this.httpService
+      .get(
+        'https://raw.githubusercontent.com/ApeSwapFinance/apeswap-yields/main/config/pools.json',
```
Comment (ape-swap) [inline src/stats/stats.service.ts:832]: Lets make this configurable.

File: src/stats/stats.service.ts [hunk 2]
```diff
@@ -823,4 +820,44 @@ export class StatsService {
...
+      stakeToken: pool.stakingToken.address[this.chainId],
+      stakeTokenIsLp: pool.stakingToken.lpToken,
+      rewardToken: pool.rewardToken.address[this.chainId],
+      rewardPerBlock: pool.rewardPerBlock,
+      startBlock: pool.startBlock,
+      bonusEndBlock: pool.bonusEndBlock,
+      abi: this.getABI(pool.abi),
+    }));
+
+    await this.cacheManager.set('pools', pools, { ttl: 300 });
```
Comment (ape-swap) [inline src/stats/stats.service.ts:849]: Do we need cache on this layer? 

If so maybe we can lower it so we can test the API quicker. As all the caches start to add up

**Review by ape-swap — CHANGES_REQUESTED** [review]
Great work!

Remember to remove the old code. I also left some comments

File: src/stats/stats.service.ts (hunk 1 above)
Comment (ixm-759) [reply src/stats/stats.service.ts]: An environment variable was added, it must be added in each POOL_LIST_URL environment

File: src/stats/stats.service.ts (hunk 2 above)
Comment (ixm-759) [reply src/stats/stats.service.ts]: The cache was removed, since at the end of the day the method is inside another in which cache is already applied.

**Pull request merged**